packages/eslint-plugin/docs/rules/no-dupe-class-members.md

```diff
@@ -6,5 +6,11 @@ description: 'Disallow duplicate class members.'
 >
 > See **https://typescript-eslint.io/rules/no-dupe-class-members** for documentation.
 
+:::danger
+
+The code problem checked by this ESLint rule is automatically checked by the TypeScript compiler. Thus, it is not recommended to turn on this rule in new TypeScript projects. You only need to enable this rule if you prefer the ESLint error messages over the TypeScript compiler error messages. (When this rule was originally written, the TypeScript compiler checks did not exist.)
+
+:::
```

Comment:
Yeah I'm not a huge fan of copypasta. At the very least, let's add a shared JSX component that the rules can embed.

It'd be even more automagical to add it to the rule metadata. Maybe a `supercededByTypeScript` property in `RuleMetaDataDocs`? cc @bradzacher & @Josh-Cena from #4156.

Reply:
I would suggest [imported Markdown](https://docusaurus.io/docs/markdown-features/react#importing-markdown) in this case

Reply:
> But TBH if it's just two rules

There's more than 2 rules, I can add the other rules that this applies to in a future PR

Reply:
Ooh what are the other rules?

Reply:
everything here: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/src/configs/eslint-recommended.ts

Reply:
notice that the two warnings that I added in this PR are slightly different, so any proposed templating mechanism to reduce DRY would need to be able to take parameters.

@Josh-Cena does imported markdown support that?

Reply:
For now, I removed the duplication with a shared JSX component.